pages/mesosphere/dcos/1.12/release-notes/1.12.5/index.md

```diff
@@ -0,0 +1,78 @@
+- Fixed DC/OS clients that do not adhere to Exhibitor TLS for versions 1.12 and 1.13. (DCOS-58361)
+- Fixed DC/OS Backup for Exhibitor TLS. (COPS-5313, DCOS-60918)
+- Added support for Mesos overlay networking to drop agents from the state. (DCOS_OSS-5536)
+- DC/OS now supports multi-role framework in the authorization module. (DCOS-54635)
+- The time it takes to deploy a service with a L4-VIP has been reduced by pruning VIPs, which reduced growth of state messages exchanged among dcos-net processes.  (DCOS-56071, DCOS_OSS-5356)
+- You can now delete a service endpoint via Form. (DCOS-21472) 
+- You can now save Docker parameters, as expected. (DCOS-45226)
+- Fixed TypeError in ResourcesUtil.getAvailableResources (DCOS-46307)
+- Error message for invalid input in package creation now goes away after updating the field. (DCOS-46413) 
+- Fixed an issue where Mesos master memory use seems to increase (DCOS-55315)
+- Fixed an issue where `dcos-signal` did not report customer ID from customer environments. (DCOS-55882)
+- Fixed issue where Secrets folder can fill up a partition. (DCOS-56481)
+- Fixed potential deadlock in RPC authenticatee (DCOS-57388)
+- Fixed Chrome issue on OS X such that certificates start on 30 June 2019. (DCOS-60205)
+- Fixed an issue where newly added public agents were not showing up in the DC/OS UI. (COPS-5606, DCOS-61274)
+- Fixed an issue where Secrets sometimes disappear from JSON editor in UI (COPS-4928, DCOS-55692)
+- Fixed an issue where the UI form deletes `networkNames` property from pod endpoints (DCOS-50452)
+- Secret generator no longer produces extra authentication warnings in Enterprise Mesos modules (COPS-4767, DCOS-52793)
+- DC/OS now supports DESTROY_RAW_DISK authorization action. (DCOS-50932)
+- Under heavy load, direct requests to the IAM could be slowed down, making the IAM slow to respond to all requests. Mesos master now uses a login endpoint that goes through Admin Router, reducing the effect of any slow senders. This addresses a rare error condition in which the IAM system can become unavailable, potentially leading to Mesos task launch errors in the strict security mode. (DCOS-56053)
+- Package specific icons are now showing up properly for all services installed from Universe. (DCOS-53674, COPS-4710) 
+- Fixed a licensing permissions error, that resulted in the following message, "error getting LicenseID. Got error: Get http://unix/licenses: dial unix /run/dcos/dcos-licensing.sock: connect: permission denied" (DCOS-57291)
+- Updated code to unify frequency between DC/OS Signal EE and OSS; signal now sends telemetry data every five minutes. (DCOS-57281)
+- Updated DC/OS master to accept the most up to date plugins. Pinned the dcos CLI plugins in every DC/OS branch for both OSS and Enterprise versions, and consolidated all dcos cli instantiations into one fixture. (DCOS-57266)
+- Completed work to unify OSS and Enterprise versions of DC/OS Signal. (DCOS-57062)
```

Comment:
Does this have some user benefit?  Otherwise it just seems like housekeeping on our end.